src/commands/synthetics/__tests__/cli.test.ts

```diff
@@ -113,8 +122,10 @@ describe('run-test', () => {
       command['configPath'] = overrideCLI.configPath
       command['datadogSite'] = overrideCLI.datadogSite
       command['failOnCriticalErrors'] = overrideCLI.failOnCriticalErrors
+      command['failOnMissingTests'] = overrideCLI.failOnMissingTests
```

Comment:
Should this be part of the same PR? Isn't it part of the ongoing refactoring for error handling?

Reply:
No it's not. It's just that this test was incomplete, and I didn't want to create a PR just to update tests.